src/internals/StreamHttpReply.hpp

```diff
@@ -81,6 +83,8 @@ class StreamHttpReply: public AbstractStreamHttpReply
 public:
     StreamHttpReply(Stream& stream, const String& contentType);
     virtual void send(const String& data, const bool gzipencoded=false) { AbstractStreamHttpReply::send(data, "OK"); };
+    virtual void send(const uint8_t* buf, const size_t size, const String& title="OK") { AbstractStreamHttpReply::send(buf, size, title); };
+    virtual void sendHeader(size_t size, const String& title="OK") { AbstractStreamHttpReply::sendHeader(size, title); }
```

Comment:
I think this can be a private method instead.

Reply:
Do you mean the new `send()` or `sendHeader()`? Before this PR, having this as a public method could let the user send a binary response like so:

```c++
StreamHttpReply res(client, content_type);
res.sendHeader(length);
client.write(buf, length);
```
Another use case I can see is sending the header and then passing `client` to a function that accepts a `Stream`. In fact, I'm doing that in my current project. Personally, I think it should be public.